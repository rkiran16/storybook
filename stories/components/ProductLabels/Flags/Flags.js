import { html } from "lit-html";

export const Flags = ({ single, title1, title2 }) => {
  if (single) {
    return html`
      <span
        class="sub-title-4 single-flag badge bg-info text-uppercase fw-normal"
        >${title1}</span
      >
    `;
  } else {
    return html`
      <span
        class="sub-title-4 badge bg-info double-flag-1 text-uppercase fw-normal"
        >${title1}</span
      >
      <span
        class="sub-title-4 badge bg-info double-flag-2 text-uppercase fw-normal"
        >${title2}</span
      >
    `;
  }
};
