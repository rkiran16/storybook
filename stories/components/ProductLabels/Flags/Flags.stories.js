import { Flags } from "./Flags";

export default {
  title: "Components/ProductLabels/Flags",
  parameters: {
    layout: "centered",
    zeplinLink:
      "https://app.zeplin.io/project/62902a463701ffb0e532df2b/screen/63053ee9c9b39414a05fab9b",
  },
};

const FlagsTemplate = (args) => Flags(args);

export const Single = FlagsTemplate.bind({});
Single.args = {
  single: true,
  title1: "Recommended",
  title2: "",
};

export const Double = FlagsTemplate.bind({});
Double.args = {
  single: false,
  title1: "New",
  title2: "Recommended",
};
