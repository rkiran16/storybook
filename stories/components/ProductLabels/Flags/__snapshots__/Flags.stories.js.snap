// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/ProductLabels/Flags Double 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
      <span
        class=\\"sub-title-4 badge bg-info double-flag-1 text-uppercase fw-normal\\"
        >",
    "</span
      >
      <span
        class=\\"sub-title-4 badge bg-info double-flag-2 text-uppercase fw-normal\\"
        >",
    "</span
      >
    ",
  ],
  "values": Array [
    "New",
    "Recommended",
  ],
}
`;

exports[`Storyshots Components/ProductLabels/Flags Single 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
      <span
        class=\\"sub-title-4 single-flag badge bg-info text-uppercase fw-normal\\"
        >",
    "</span
      >
    ",
  ],
  "values": Array [
    "Recommended",
  ],
}
`;
